### ISSUE TITLE
Fatal error during composer update with PHP 5.3

### ISSUE THREAD

**Issue by veewee** [hirak/prestissimo#16]
During composer update on PHP 5.3 the plugin crashes:

```
[17.1MB/0.15s] Loading composer repositories with package information

PHP Fatal error:  Using $this when not in object context in /home/travis/.composer/vendor/hirak/prestissimo/src/CurlRemoteFilesystem.php on line 96

PHP Stack trace:

PHP   1. {main}() /home/travis/.phpenv/versions/5.3.29/bin/composer:0

PHP   2. require() /home/travis/.phpenv/versions/5.3.29/bin/composer:25

PHP   3. Composer\Console\Application->run() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/bin/composer:43

PHP   4. Symfony\Component\Console\Application->run() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Console/Application.php:98

PHP   5. Composer\Console\Application->doRun() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/vendor/symfony/console/Application.php:123

PHP   6. Symfony\Component\Console\Application->doRun() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Console/Application.php:167

PHP   7. Symfony\Component\Console\Application->doRunCommand() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/vendor/symfony/console/Application.php:192

PHP   8. Symfony\Component\Console\Command\Command->run() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/vendor/symfony/console/Application.php:844

PHP   9. Composer\Command\UpdateCommand->execute() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/vendor/symfony/console/Command/Command.php:259

PHP  10. Composer\Installer->run() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Command/UpdateCommand.php:158

PHP  11. Composer\Installer->doInstall() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Installer.php:232

PHP  12. Composer\DependencyResolver\Pool->addRepository() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Installer.php:413

PHP  13. Composer\Repository\ComposerRepository->hasProviders() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/DependencyResolver/Pool.php:99

PHP  14. Composer\Repository\ComposerRepository->loadRootServerFile() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Repository/ComposerRepository.php:259

PHP  15. Composer\Repository\ComposerRepository->fetchFile() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Repository/ComposerRepository.php:456

PHP  16. Hirak\Prestissimo\CurlRemoteFilesystem->getContents() phar:///home/travis/.phpenv/versions/5.3.29/bin/composer/src/Composer/Repository/ComposerRepository.php:651

PHP  17. Hirak\Prestissimo\CurlRemoteFilesystem->fetch() /home/travis/.composer/vendor/hirak/prestissimo/src/CurlRemoteFilesystem.php:97

PHP  18. Hirak\Prestissimo\{closure}() /home/travis/.composer/vendor/hirak/prestissimo/src/CurlRemoteFilesystem.php:148
```

More information:
https://travis-ci.org/phpro/grumphp/jobs/105100866

**Comment by hirak** [comment]
@veewee  I released 0.1.5. Prease try new version.

```bash
$ composer global remove hirak/prestissimo
$ composer global require hirak/prestissimo
```

**Comment by veewee** [comment]
Nice, thanks!
